notebooks/VectorSearch_QuestionRetrieval.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p>I wonder if we shold be creating a separate section in <code>depencies.yaml</code> that will output a new conda environment yaml which we can simply tell the user "please create a new conda environment using this yaml file and install raft to run this notebook". That would make it much easier for a user to go from zero to notebook example.</p><p> </p><p>Can we separate the imports from the loading of the data? I've found it to be a good practice as a sanity check so users can quickly see if they have all the needed packages. Another reason this will be useful is that we can document the actual data loading cell as such, while still keeping the general introduction and description separate. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I think we should go to the simplest option. How about adding a cell with
`!pip install sentence_transformers torch pylibraft-cu11 --extra-index-url=https://pypi.nvidia.com`?
This will minimize user interaction and it will work out of the box.

Reply:
I think to get this (and the IVF-PQ notebook from @achirkin) into 23.08, we should go ahead and do a pip install for now (let's just point to the raft documentation for raft install and have the cell do a `pip install sentence_transformers torch` for now). We can follow-up with this at a later time.